tendrl/node_agent/collectd/collectors/ini2json.py

```diff
@@ -0,0 +1,126 @@
+from ConfigParser import ConfigParser
```

Comment:
Move this

and this https://github.com/Tendrl/gluster-integration/blob/master/tendrl/gluster_integration/ini2json.py 

to commons and re-use